product_docs/docs/biganimal/release/getting_started/docker_env.mdx

```diff
@@ -0,0 +1,176 @@
+---
+title: Installing PostgreSQL in a Docker container on your local machine
+navTitle: Installing PostgreSQL in a Docker container
+description: Learn how to install PostgreSQL in a Docker container on your local machine for development purposes.
+deepToC: true
+---
+
+Using Docker for your local PostgreSQL development environment streamlines setup, ensures consistency, and simplifies management. It provides a flexible, isolated, and portable solution that can adapt to various development needs and workflows.
+
+## Prerequisites
+
+* Docker-compatible OS (macOS, Windows, Linux)
+
+## Preparing Docker
+
+### Install Docker: 
+
+* macOS: Download and install Docker Desktop from [Docker’s official website](https://docs.docker.com/desktop/install/mac-install/).  
+* Windows: Download and install Docker Desktop from [Docker’s official website](https://docs.docker.com/desktop/install/windows-install/). Ensure WSL 2 is enabled if using Windows 10 or later.  
+* Linux: Install Docker using your distribution’s package manager. For example, on Ubuntu:
```

Comment:
This may be better as a pointer to the docs as the example installs a no longer official docker.

Reply:
fixed it.